### PR TITLE
For #28468, address feedback for login gui

### PR DESCRIPTION
**Pull request by jfboismenu-adsk**
- Textbox is now fully disabled when not editable
- When session is expired, message is no longer is red but in gray.
- Focus is appropriately set to the next available empty QLineEdit

### CODE REVIEW

**Comment by manneohrstrom** [comment]
Here's a screenshot with disabled fields:

image

File: python/tank_vendor/shotgun_authentication/login_dialog.py [hunk 1]
```diff
@@ -48,24 +48,28 @@ def __init__(self, is_session_renewal, hostname="", login="", fixed_host=False,
...
-        else:
-            self.ui.site.setFocus()
-
         # set the logo
         self.ui.logo.setPixmap(QtGui.QPixmap(":/shotgun_authentication/shotgun_logo_light_medium.png"))
 
+        if fixed_host:
+            self._disable_widget(
+                self.ui.site,
+                "You can't edit the host, your core is configured for this site."
```
Comment (manneohrstrom) [inline python/tank_vendor/shotgun_authentication/login_dialog.py]: This seems a little technical - maybe it should say "The shotgun site has been predefined and cannot be modified" or something?

**Comment by manneohrstrom** [comment]
One tiny comment, otherwise looking great!

File: python/tank_vendor/shotgun_authentication/login_dialog.py (hunk 1 above)
Comment (jfboismenu-adsk) [inline python/tank_vendor/shotgun_authentication/login_dialog.py]: It does sound better. Thanks!

**Pull request closed without merging**